packages/x-charts/src/internals/plugins/featurePlugins/useChartCartesianAxis/zoom.types.ts

```diff
@@ -2,7 +2,20 @@
 import { AxisId } from '../../../../models/axis';
 import type { ExtremumFilter } from './useChartCartesianAxis.types';
 
-export type ZoomData = { axisId: AxisId; start: number; end: number };
+export type ZoomData = {
+  /**
+   * The axis id of the zoomed axis.
+   */
+  axisId: AxisId;
+  /**
+   * The starting percentage of the zoom range. In the range of 0 to 100.
```

Comment:
Not sure if is makes sense to mention this

```suggestion
   * The starting percentage of the zoom range. In the range of 0 to 100, inclusive on both ends.
```

Reply:
Why would it be exclusive?

Reply:
Just wanted to make sure it was clear. I guess it makes sense that it is inclusive, even if we don't specify it, but it might help clear confusion. For example, `Math.random()` is inclusive of 0, but exclusive of 1 ([source](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random)). 

Not a big deal, though. Up to you